Mopy/mash/mosh.py

```diff
@@ -6265,8 +6265,11 @@ def remapObject(self,object,newIMod,newIObj=-1):
         newObject = (newIMod,newIObj)+object[2:]
         if objRecords and objRecords[0].name == 'MVRF':
             data = cStringIO.StringIO()
-            data.write(struct.pack('i',newIObj)[:3])
-            data.write(struct.pack('B',newIMod))
+            if newIMod > 255:
+                data.write(struct.pack('2i', (newIMod, newIObj)))
+            else:
+                data.write(struct.pack('i',newIObj)[:3])
+                data.write(struct.pack('B',newIMod))
```

Comment:
@NullCascade @Greatness7 
Is there a reason to be inside the if/else statement? Shouldn't it be outside to catch both situations?:
==> data.write(struct.pack('B',newIMod))